automation/check-patch.e2e-multus-functests.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -xe
+
+# This script should be able to execute multus
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-multus-functests.sh
+
+teardown() {
+    make cluster-down
+    rm -rf "${TMP_COMPONENT_PATH}"
```

Comment:
in case the above command failed, this one wont be reached, so maybe u want to reorder ?

Reply:
Done